src/main/java/pl/wrapper/parking/infrastructure/exception/GlobalExceptionHandler.java

```diff
@@ -11,46 +12,51 @@
 
 @ControllerAdvice
 @RequiredArgsConstructor
+@Slf4j
 class GlobalExceptionHandler {
 
     @ExceptionHandler(Exception.class)
-    public ResponseEntity<ErrorWrapper> handleGeneralException(Exception e, HttpServletRequest request) {
+    public ResponseEntity<ErrorWrapper> handleGeneralException(HttpServletRequest request) {
         HttpStatus status = HttpStatus.INTERNAL_SERVER_ERROR;
-        ErrorWrapper errorWrapper =
-                new ErrorWrapper("An error has occured", status, request.getRequestURI(),status);
+        String message = "An error has occured";
+        ErrorWrapper errorWrapper = new ErrorWrapper(message, status, request.getRequestURI(), status);
+        log.error(message);
```

Comment:
Logging exceptions is so that we can see where they occurred. Leave the Exception argument and change: log.error(message) -> log.error(message + " at: " + uri + "; Details: " + e.getMessage()) so that we know which endpoint and what exactly failed